19_pascal/solution/pascal-solution.js

```diff
@@ -0,0 +1,31 @@
+const pascal = function (counter, currentLine = [1]) {
+  if (counter === 1) return currentLine;
+
+  const halfOfNextLine = currentLine.reduce(
+    (accumulator, currentElement, index, originalArray) => {
+      const nextElement = originalArray[index + 1]
+      if (currentElement <= nextElement) {
+        return [...accumulator, currentElement + nextElement];
+      }
+      return accumulator;
+    },
+    [1],
+  );
```

Comment:
Can we come up with a more descriptive name than `accumulator`?
It would help people reading to understand what that accumulator accumulates i.e. what it represents.

Reply:
What do you think about `otherElements`?

Reply:
Since we're dealing with numbers here and aren't concerned with "opening" the solution up to generic non-number use, I'd propose changing the params to `(numbers, currentNumber, index)`, then the `nextElement` variable could become `nextNumber` for consistency.

Given that we haven't extracted out a helper function for the `reduce` callback for use with multiple arrays, we wouldn't really need the 4th param and referencing `currentLine` might be more intuitive for learners reading it.

Reply:
Any issues with the above?

Reply:
- used better names for variables